tests/artist-create.test.js

```diff
@@ -0,0 +1,28 @@
+// tests/artist-create.js
```

Comment:
Theres no need for the commented name of the file here, it just creates extra noise so it should be removed. This applies for all other uses of this throughout the review